src/test/java/com/salesforce/dataloader/client/PartnerClientTest.java

```
@@ -214,11 +214,20 @@ public void testDescribeSObjects() throws Exception {
 
         int numDescribes = 0;
         for (String objectType : client.getDescribeGlobalResults().keySet()){
-            DescribeSObjectResult describeResult = client.describeSObject(objectType);
-            numDescribes++;
-            assertNotNull(describeResult);
-            assertEquals(objectType, describeResult.getName());
-            assertEquals(numDescribes, client.getEntityDescribeMap().size());
+        	try {
+        		DescribeSObjectResult describeResult = client.describeSObject(objectType);
+        		numDescribes++;
+                assertNotNull(describeResult);
+                assertEquals(objectType, describeResult.getName());
+                assertEquals(numDescribes, client.getEntityDescribeMap().size());
+        	} catch (Exception ex) {
+        		if (ex.getMessage().contains("jsonNot")) {
+        			System.out.println("PartnerClient.testDescribeSObjects: Unable to call describeSObject for " + objectType);
```

I think it's better to throw an exception and include the custom message.